tools/coding_standard/README

```diff
@@ -6,3 +6,8 @@ whitespace.py    detects TAB characters and trailing whitespace
 homepage.py      detects outdated LAMMPS homepage URLs (pointing to sandia.gov instead of lammps.org)
 errordocs.py     detects deprecated error docs in header files
 versiontags.py   detects .. versionadded:: and .. versionchanged:: with pending version date
+
```

Comment:
Even simpler is going to the "src" folder and typing "make check", or one of the possible "make check-XXX" or "make fix-XXX" commands. E.g. "make fix-whitespace" to remove known whitespace issues.
Please see: https://docs.lammps.org/Build_development.html#coding-style-utilities and https://docs.lammps.org/Tools.html#coding-standard